install.sh

```diff
@@ -209,16 +200,14 @@ check_deps() {
 }
 
 check_python() {
-  if type -P python2.7 &> /dev/null; then
-    PYTHONVERSION="2.7"
-    PYTHON="python2.7"
+  if type -P python3 &> /dev/null; then
+    PYTHON="python3"
   elif type -P python &> /dev/null; then
-    PYTHONVERSION=`python -c 'import sys; print(".".join(map(str, sys.version_info[:2])))'`
     PYTHON="python"
   fi
-
-  if [[ $PYTHONVERSION != "2.7" ]]; then
-    echo "Python version 2.7 is required to install pty.js. Please install python 2.7 and try again. You can find more information on how to install Python in the docs: https://docs.aws.amazon.com/cloud9/latest/user-guide/ssh-settings.html#ssh-settings-requirements"
+  
+  if ! type -P "$PYTHON" &> /dev/null; then
+    echo "Python version is required to install pty.js. Please install python and try again. You can find more information on how to install Python in the docs: https://docs.c9.io/ssh_workspaces.html"
```

Comment:
`version` reads a bit awkward here (2nd word). Can we remove it? Also, I think we should keep the original url.

```suggestion
    echo "Python is required to install pty.js. Please install python and try again. You can find more information on how to install Python in the docs: https://docs.aws.amazon.com/cloud9/latest/user-guide/ssh-settings.html#ssh-settings-requirements"
```